projectile.el

```diff
@@ -1250,10 +1250,10 @@ Only buffers not visible in windows are returned."
 
 (defun projectile-normalise-paths (patterns)
   "Remove leading `/' from the elements of PATTERNS."
-  (-non-nil (--map (and (string-prefix-p "/" it)
-                        ;; remove the leading /
-                        (substring it 1))
-                   patterns)))
+  (-non-nil (mapcar (lambda (it) (and (string-prefix-p "/" it)
```

Comment:
It'd rename all those `it` arguments to something meaningful. This name was required for `--something` macros but we can certainly do better.